test/utils/image/manifest.go

```diff
@@ -65,6 +67,7 @@ func initReg() RegistryList {
 	registry := RegistryList{
 		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
 		DockerLibraryRegistry:   "docker.io/library",
+		DockerHumbleC:           "docker.io/humblec",
```

Comment:
@humblec can you point us to a public registry where we can pull the gluster provisioner image from?

Reply:
@msau42 @johnSchnake  while developing the e2e I was trying to use `gcr` or some other common repo , but somehow I didnt have the push access. Is there a common repo path where I can push this image ? If yes, please share it, so thatI can move or copy this container to that common path and update it via PR.

Reply:
@spiffxp Could you comment on the issue above?

I'm not sure really where the image comes from, but I would assume he needs to put any code to build the image into `test/images` then just ask someone in <some channel> to bulid/push it?

Reply:
/hold

we *cannot* be adding personal repos!

Reply:
@humblec please work with @jsafrane and @childsb to push your image to the quay repo for kubernetes-incubator/external-storage projects

Reply:
@dims just to be clear, we all plan on getting away from this repo. The issue is that apparently this has been in for a while now and, since it didnt go through imageutils, no one else knew.

IMO this PR should go in just as-is to centralize things and a separate issue/PR to _fix_ what this PR exposed.

Reply:
@jsafrane / @childsb can you please give me access to external storage quay repo ? I thought I have access but pushing is failing :
```
[root@localhost tmp]# docker login quay.io -u humble
Password: 
WARNING! Your password will be stored unencrypted in /root/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded

[root@localhost tmp]# docker push quay.io/external_storage/glusterdynamic-provisioner:v1.0
The push refers to repository [quay.io/external_storage/glusterdynamic-provisioner]
941d194d7754: Preparing 
427bb0cd8006: Preparing 
ccc59abfdd55: Preparing 
60c88ea6c162: Preparing 
d66daa39c937: Preparing 
53e6d99f7596: Waiting 
unauthorized: access to the requested resource is not authorized
[root@localhost tmp]# 
```

Reply:
@johnSchnake i'd rather revert https://github.com/kubernetes/kubernetes/pull/59879 than let this in.

Reply:
Its been there for 1.5y so it would just seem strange to me to revert; I guess I'm less familiar with reverting things like that which would affect numerous versions and may impact other PRs that have gone in during that time.

It seems like we just need to fix what is there but I don't know what precedents exist. Also makes it clear that could use some automation to aid on this front. Creating a ticket about it.

Reply:
https://github.com/kubernetes/kubernetes/issues/81169

Reply:
@johnSchnake i extremely dislike both options for sure. of the two, i don't want it to end up in `test/utils/image/manifest.go` for sure, also i was comparing that to the option of just removing all the code that references that image.

cc @msau42 @saad-ali

Reply:
@dims the code is there for long time. As mentioned in my previous comment if anyone can give access to common repo or path, I can push the image and change the code without any delay. The only reason to keep it in private repo was this.

Reply:
@humblec why not say https://hub.docker.com/u/gluster ? (in other words, why should it be in the k8s gcr repo?)

Reply:
All the external-provisioners developed under kubernetes-incubator/external-storage are using quay.io/external-storage to store their images.  I think that is the most obvious place to put the gluster image

Reply:
@msau42 working with Jan and Brad on this , Hopefully this will be sorted out soon.

Reply:
Looks like @childsb is on pto. Jan dont have access of the repo. @wongma7 is it something you can help ?

Reply:
@msau42 @johnSchnake @dims  Discussed with @childsb and as per him,  it is better to host it under `gluster` org in docker hub. I have moved the image to `gluster` org and updated the code by https://github.com/kubernetes/kubernetes/pull/82115.